data/tlg0077/tlg001/tlg0077.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,1709 @@
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Stadiasmus Magni Maris</title>
+<author>Anonymous</author>
+<editor xml:lang="lat">Carolus Müllerus</editor>
+<funder>Harvard Library Arcadia Fund</funder>
+<principal>Gregory Crane</principal>
+<respStmt>
+<persName xml:id="DDD">Digital Divide Data</persName>
+<resp>Corrected and encoded the text</resp>
+</respStmt>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>Editor-in-Chief, Perseus Digital Library</resp>
+</respStmt>
+<respStmt>
+<persName>Matt Munson</persName>
+<resp>Project Manager (University of Leipzig), 2016 - present</resp>
+</respStmt>
+<respStmt>
+<persName>Annette Gessner</persName>
+<resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+</respStmt>
+<respStmt>
+<persName>Thibault Clérice</persName>
+<resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+</respStmt>
+<respStmt>
+<persName>Bruce Robertson</persName>
+<resp>Technical Advisor</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+
+<idno type="filename">tlg0077.tlg001.1st1K-grc1.xml</idno>
+<availability>
+<p>Available under a Creative Commons Attribution-ShareAlike 4.0 International License</p>
+</availability>
+<date>2017</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+<biblStruct>
+<monogr>
+
+
+<title xml:lang="lat">Geographi Graeci Minores</title>
+<editor>
+<persName>
+<name xml:lang="lat">Carolus Müller</name>
+<addName xml:lang="deu">Karl Müller</addName>
+</persName>
+</editor>
+<author ref="urn:cts:greekLit:tlg0077">Anonymous</author>
+
+<imprint>
+<publisher>Ambroise-Firmin Didot</publisher>
+<pubPlace>Paris</pubPlace>
+<date>1855</date>
+</imprint>
+
+<biblScope unit="volume">1</biblScope>
+
+</monogr>
+<ref target="https://archive.org/details/geographigraeci00philgoog">The Internet Archive</ref>
+</biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+
+
+<encodingDesc>
+
+<p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE
+Architecture.</p>
+
+
+<refsDecl n="CTS">
+
+<cRefPattern n="paragraph" matchPattern="(.+).(.+)"
+replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:p[@n='$2'])">
+<p>This pointer pattern extracts paragraphs</p>
+</cRefPattern>
+
+<cRefPattern n="section" matchPattern="(.+)"
+replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+<p>This pointer pattern extracts sections</p>
+
+</cRefPattern>
+
+
+</refsDecl>
+
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
```

Comment:
Since you have the "deu" xml:lang value above, you should also define that here.